stories/5-components/Links/AuLinkExternal.stories.js

```diff
@@ -61,3 +67,64 @@ Component.args = {
   width: '',
   active: false,
 };
+
+export const Secondary = Template.bind({});
+Secondary.args = {
+  text: 'External link',
+  href: 'https://www.vlaanderen.be/',
+  route: 'index',
+  skin: 'secondary',
+  icon: 'manual',
+  iconAlignment: 'left',
+  hideText: false,
+  width: '',
+  active: false,
+};
+Secondary.parameters = {
+  chromatic: { disableSnapshot: true },
```

Comment:
Is there a specific reason why we don't want Chromatic to snapshot these variants?

Reply:
This was mostly because of a previous comment (in another PR): https://github.com/appuniversum/ember-appuniversum/pull/319#discussion_r967063270

If you think this wont impact the runtime (of the tests) too much, we can opt to enable them again?

Reply:
I would honestly enable it regardless if it impacts the test time. It seems like a good thing to test visually since we can't really verify if it works through other tests. And I would be surprised if it had _that_ much impact?

Reply:
The impact was more regarding the future if the grand total of components (and thus tests) gets larger. More like a mindset to not test everything in absolute detail.

But I also agree to your point of view. Maybe we'll enable the tests here (and for `AuLink`) again, but keep an eye out if the runtime gets too long (if that is even possible 😅)?

Reply:
Yea, I'm not super familiar with Chromatic but I can't find how long it took to verify all the snapshots. Only the build time but that's only 15 seconds.

In any case, I think will always be the slowest part of our CI so I don't think we need to be conservative with Chromatic for now. We might need to look into it if we start into the free plan limitations but I don't think that will happen soon.